userguide/plugins.rst

```diff
@@ -160,8 +160,10 @@ repository, update the plugin jail by navigating to the
 :menuselection:`Jails` page and clicking |ui-options| and
 :guilabel:`Update`.
 
-.. note:: Plugins are updated using :samp:`iocage update {PLUGIN}`
-   and upgraded using :samp:`iocage upgrade {PLUGIN}`.
+.. note:: Plugins can also be updated using :samp:`iocage update {PLUGIN}`
```

Comment:
I don't think this needs to be a note.  Instead, break it into two paragraphs: one that describes what an update is and how it is done, and a second that describes what an upgrade is and how to do it.